File: 3-Bimestre/ex-5.py
contador = 1
n = int(input('Digite um número de 1 a 10: '))

if n >= 1 and n <= 10:
    while contador <= 10:
        print(f'{n} x {contador} = {n * contador}')
        contador += 1
else:
    print('Número inválido')

File: 3-Bimestre/C++/ex-5.cpp
#include <iostream>
using namespace std;

int main() {
    int n, contador = 1;
    cout << "Digite um numero de 1 a 10: ";
    cin >> n;

    if (n >= 1 && n <= 10) {
        while (contador <= 10) {
            cout << n << " x " << contador << " = " << n * contador << endl;
            contador++;
        }
    } else {
        cout << "Numero invalido";
    }

    return 0;
}
